vllm/config/__init__.py

```diff
@@ -86,6 +86,21 @@
 logger = init_logger(__name__)
 DataclassInstanceT = TypeVar("DataclassInstanceT", bound=DataclassInstance)
 
+
+def _validate_quantization_method(value: Any) -> Any:
+    """Shared validation logic for quantization method strings.
+    
+    Handles empty strings, dots, and converts to lowercase.
+    """
+    if isinstance(value, str) and value.strip() == "":
+        return None
+    if isinstance(value, str) and value.strip() == ".":
+        return None
+    if isinstance(value, str):
+        return value.lower()
+    return value
```

Comment:
This should not be necessary, empty strings are already handled in the CLI by https://github.com/vllm-project/vllm/blob/f4962a6d55a340ebb569d377c842deff7611d8f7/vllm/engine/arg_utils.py#L81-L90

If empty strings are being passed via the Python API then this is incorrect usage.